controlplane/kubeadm/config/default/manager_arguments.yaml

```diff
@@ -0,0 +1,4 @@
+- op: replace
+  path: "/spec/template/spec/containers/0/args"
+  value:
+    - "--enable-leader-election"
```

Comment:
Same as above, this should be the default value and we shouldn't need to override it here with a json patch